manifests/container/config.pp

```diff
@@ -0,0 +1,71 @@
+
+# TODO: ssl_enabled - probably use another parameter, e.g. insecure
+# TODO: to check controller_virtual_ip, glance_api_ip, neutron_metadata_ip
+# TODO: pass from upper layer: configdb_cassandra_user, configdb_cassandra_password
+# TODO: pass from upper layer: analyticsdb_cassandra_user, analyticsdb_cassandra_password
+
+define contrail::container::config (
```

Comment:
Plan here is to be at-par with contrail-ansible with respect to exposing variables (changes will need to be made to upper layers also)